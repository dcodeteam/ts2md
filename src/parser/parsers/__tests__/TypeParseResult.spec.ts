import { testParser } from "../../../__tests__/test-utils";

testParser("type");
