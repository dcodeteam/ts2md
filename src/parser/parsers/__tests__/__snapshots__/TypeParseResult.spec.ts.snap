// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`type type-base.ts 1`] = `
ProjectParseResult {
  entryModule: type/type-base.ts,
  modules: Map {
    type/type-base.ts => ModuleParseResult {
      imports: Array [],
      modulePath: type/type-base.ts,
      nodes: Array [
        TypeParseResult {
          defaultExported: false,
          documentation: This is type docs.,
          documentationTags: Array [],
          exported: true,
          id: TypeBase,
          type: "TYPE" | "BASE",
        },
      ],
      reexports: Array [],
    },
  },
}
`;
